.github/workflows/main.yml

```diff
@@ -31,9 +31,27 @@ jobs:
           node-version: '12.x' # TODO: match nvmrc version
       - name: Install dependencies
         run: yarn --frozen-lockfile
+      - name: Setup sonar scanner
+        uses: warchant/setup-sonar-scanner@v1
       - name: Lint
         run: yarn lint
       - name: Run tests
         run: |
           echo TYPEORM_PASSWORD=${{ secrets.TEST_POSTGRES_PASSWORD }} >> .env.ci
           yarn test:ci
+      # Another existing action, this one to checkout the repository
+      - name: "Checkout repository on branch: ${{ github.REF }}"
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.REF }}
+          fetch-depth: 0
```

Comment:
Should we do this again? I think the code is already checked, but maybe I'm missing sth

Reply:
not sure, sonar uses the commit hash to tag the analysis, but I will try to remove it and see what happens